deploy/backplane/mtsre/managed-odh/10-mtsre-odh-admins.SubjectPermission.yml

```diff
@@ -1,12 +1,12 @@
 apiVersion: managed.openshift.io/v1alpha1
 kind: SubjectPermission
 metadata:
-  name: backplane-mtsre-admins
+  name: backplane-mtsre-managed-odh-admins
```

Comment:
Any reason to change the name here? We'd preferably keep the name the same across changes.

Reply:
+1, would rather keep the same name

Reply:
Hi folks, as I mentioned, in the other addons's backplane RBAC as well, the naming convention of `backplane-mtstre-<addon name>-admins` is being followed, that's why I suggested this name change here.
- Reference addon - https://github.com/openshift/managed-cluster-config/blob/ba5ac3ef87b276502c85e450c32283134659a1b5/deploy/backplane/mtsre/reference-addon/10-mtsre-reference-addon-admins.SubjectPermission.yml#L4
- OCS Converged addon - https://github.com/openshift/managed-cluster-config/blob/ba5ac3ef87b276502c85e450c32283134659a1b5/deploy/backplane/mtsre/ocs-converged/10-mtsre-ocs-converged-admins.SubjectPermission.yml#L4